openedx/features/pakx/lms/overrides/utils.py

```diff
@@ -1,16 +1,18 @@
 """ Overrides app util functions """
 
+import re
 from datetime import datetime
 from logging import getLogger
-from re import compile, findall
```

Comment:
Why did we revert this?

Reply:
conflicts with python's built-in function name, plus clear indication of using the re module for better readbility.